adaptive_navigation/lib/scaffold.dart

```diff
@@ -221,9 +232,15 @@ class AdaptiveNavigationScaffold extends StatelessWidget {
   }
 
   Widget _buildNavigationRailScaffold() {
-    final railDestinations =
-        destinations.sublist(0, math.min(destinations.length, 7));
-    final drawerDestinations = destinations.length > 7 ? destinations : [];
+    final int railDestinationsOverflow = 7;
+    final railDestinations = destinations.sublist(
+      0,
+      math.min(destinations.length, railDestinationsOverflow),
+    );
+    final drawerDestinations = destinations.length > railDestinationsOverflow
+        ? destinations.sublist(
+            overflowDestinationAddedToMenu ? railDestinationsOverflow : 0)
```

Comment:
This would be inverted with new flag name, which should default to true

Reply:
Done